README.md

```diff
@@ -1,2 +1,335 @@
+  either by stacking them in the view,
+  or queueing them and displaying sequentially.
+
+A broader goal of the standard toast is to provide a base for more opinionated or featureful toast libraries to layer on top of.
+It will be designed and built highly extensible, 
+so library implementations can focus on providing more specific styling, better framework support, or more opinionated default.
+The intent is that any developer looking to use a toast in their work will use a standard toast,
+or a library which provides a wrapper on top of standard toast.
+
+TODO([#14](https://github.com/jackbsteinberg/std-toast/issues/14)): create an example of this layering and link to it here.
+
+## Proposed API
+
+The element is provided as a [built-in module](https://github.com/tc39/proposal-javascript-standard-library/blob/master/README.md),
+named `"std:elements/toast"`.
+
+### The `<std-toast>` element
+
+#### Attributes
+
+- [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
+- `open`: a boolean attribute, determining whether the toast is visible or not (according to the default styles). By default toasts are not shown.
+- `theme`: one of `"default"`, ???, or ???, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided)
+    - TODO([#18](https://github.com/jackbsteinberg/std-toast/issues/18)): decide on list of themes to natively support and create and style them.
+- `position`: one of `"top-left"`, `"top-center"`, `"top-right"`, `"middle-left"`, `"middle-center"`, `"middle-right"`, `"bottom-left"`, `"bottom-center"`, or `"bottom-right"`.
+The default (if the attribute is omitted or set to an invalid value) is ???.
+    - TODO([#13](https://github.com/jackbsteinberg/std-toast/issues/13)): should this positioning be an attribute or a style
+- `closebutton`: a boolean attribute, determining whether an explicit close button is shown. 
+By default toasts do not have a close button.
+    - TODO: where should the `closebutton` show up relative to toast content,
+    and how customizable should it be?
+
+All attributes will be reflected as properties on the element's JavaScript interface.
+For example:
+
+```js
+const toast = document.createElement('std-toast');
+console.log(toast.open); // false
+```
+
+#### Contents
+
+The `<std-toast>` element can be used as a generic container,
+but will work best (in terms of default styling and events) if the developer conforms to the following content model:
+
+- The first child node,
+  typically either a text node or a container like `<p>`,
+  provides a message.
+- If more than one child node is present,
+  then the last child element,
+  which can be either an `<a>` or a `<button>`,
+  provides a call to action.
+
+TODO([#17](https://github.com/jackbsteinberg/std-toast/issues/17)): what about title or icon? They should potentially also be accomodated, in a similar fashion.
```

Comment:
I think we should create and link to a tracking issue for whether this setup is the best one, to give us a place to discuss and explore alternatives. (For example, using `<button slot="action">...</button>`, or `<toastaction><button>...</button></toastaction>`, or similar.)

Reply:
Addressed in #21